packages/adapter-dynamodb/package.json

```
@@ -1,19 +1,30 @@
 {
   "name": "@next-auth/dynamodb-adapter",
   "repository": "https://github.com/nextauthjs/next-auth",
-  "version": "1.2.0",
+  "version": "2.0.0",
   "description": "AWS DynamoDB adapter for next-auth.",
   "keywords": [
     "next-auth",
     "next.js",
     "oauth",
     "dynamodb"
   ],
+  "type": "module",
+  "types": "./index.d.ts",
   "homepage": "https://authjs.dev",
   "bugs": {
     "url": "https://github.com/nextauthjs/next-auth/issues"
   },
-  "main": "dist/index.js",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./index.js"
+    },
+    "./utils": {
+      "types": "./utils.d.ts",
+      "import": "./utils.js"
+    }
```

```suggestion
    }
```

I don't think this needs to be exposed until there is a use case.

Related: https://github.com/nextauthjs/next-auth/issues/6582#issuecomment-1416010063

---

Honestly, https://github.com/nextauthjs/next-auth/blob/main/packages/adapter-dynamodb/src/utils.ts is fairly short, so we should just move it to `index.ts`.

---

Agree, let's move it to `index.ts` 👍
The error was due to the import - if I change the import to `./util.js` then it works